src/logger.ts

```diff
@@ -0,0 +1,135 @@
+import { format } from 'util';
+
+// safely preserve unpatched console.* methods in case of compat require
```

Comment:
Will these be parsed into the reference by our TypeDoc script?  If so, suggest "//Safely . . . require." with init cap and period.

Reply:
Nope.